src/Microsoft.Data/DataFrameColumn.BinaryOperations.cs

```diff
@@ -12,96 +12,248 @@ namespace Microsoft.Data
 {
     public abstract partial class DataFrameColumn
     {
+        public static DataFrameColumn Add<T>(T value, DataFrameColumn right)
```

Comment:
I'm confused why we have both public static `Add(T, DataFrameColumn)` and `DataFrameColumn.RAdd(T)`? Why do they both need to be public?

Reply:
I decided to get rid of the static methods. The static operators now call the appropriate instance methods directly.